fcsparser/api.py

```diff
@@ -27,11 +27,39 @@
 
 def fromfile(file, dtype, count, *args, **kwargs):
     """Wrapper around np.fromfile to support any file-like object."""
+    
+    dtypes = dtype.split(',')
+    field_width = []
+    
+    for dt in dtypes:
+        num_bytes = int(dt[2:])
+        field_width.append(num_bytes)
+        
```

Comment:
Can you calculate the sum once and store in a proxy variable -- it'll make reading the code easier

```
total_bytes = sum(field_width)
```

`field_width` is not the name of an iterable but a scalar, consider using pluralization